src/OrchardCore.Modules/OrchardCore.Redis/HealthChecks/RedisHealthCheck.cs

```diff
@@ -44,12 +47,12 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
             }
             else
             {
-                return HealthCheckResult.Unhealthy();
+                return HealthCheckResult.Unhealthy(description: S["There's an issue in the Redis connection."]);
             }
         }
-        catch
+        catch (Exception ex)
         {
-            return HealthCheckResult.Unhealthy();
+            return HealthCheckResult.Unhealthy(description: ex.Message);
```

Comment:
Pass the whole exception instead.

```suggestion
            return HealthCheckResult.Unhealthy("Retrieving the status of the Redis service failed.", ex);
```